pkg/metrics/metrics.go

```diff
@@ -1414,9 +1415,9 @@ func NewBPFMapPressureGauge(mapname string, threshold float64) *GaugeWithThresho
 }
 
 func init() {
-	MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{Namespace: Namespace}))
-	// TODO: Figure out how to put this into a Namespace
-	// MustRegister(prometheus.NewGoCollector())
+	MustRegister(collectors.NewProcessCollector(collectors.ProcessCollectorOpts{Namespace: Namespace}))
+	MustRegister(collectors.NewGoCollector())
```

Comment:
in my opinion, this kind of metrics is not required to be namespaced,

AFAIK, prometheus namespace is merely just a prefix, it's making sense to have namespace/prefix for this kind of metrics.  Happy to enable it back 👍.

Reply:
Agreed, the go metrics and build info metrics are nice to aggregate across different services, so I didn't think it needed namespacing either.